spec/index.md

```diff
@@ -3277,6 +3283,7 @@ S = {
   certified_data: CanisterId ↦ Blob;
   canister_history: CanisterId ↦ CanisterHistory;
   canister_log_visibility: CanisterId ↦ CanisterLogVisibility;
+  canister_log_allowlist: CanisterId ↦ Set Principal;
```

Comment:
I wouldn't introduce a new mapping `canister_log_allowlist` here since `canister_log_visibility` contains all the necessary information. This suggestion then also makes the question how to update this mapping obsolete. Please see my suggestion below on how to formulate the condition if a request sender is authorized to fetch the canister logs.

Reply:
Ok, removed `canister_log_allowlist`.